.github/workflows/publish_images_on_push.yaml

```diff
@@ -46,3 +46,18 @@ jobs:
       image_tags: ${{ github.sha }},latest
       git_ref: ${{ github.sha }}
     secrets: inherit
+
+  cloud-provider-plguins:
+    uses: ./.github/workflows/cloud_provider_plugins_build_and_push.yaml
+    with:
+      image_tags: ${{ github.sha }}
+      git_ref: ${{ github.sha }}
+    secrets: inherit
+
+  caa-peerpod-ctrl-with-cloud-provider-plguins:
```

Comment:
Same typo here: `caa-peerpod-ctrl-with-cloud-provider-plugins`

Reply:
This was deleted in a later commit, so should be squashed for clarity

Reply:
squashed